model_mlp.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import print_function
+
+import utils
+
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation
+from keras.optimizers import SGD, Adam, RMSprop
+
+import numpy as np
+np.random.seed(1337)  # for reproducibility
+
+batch_size = 128
+nb_epoch = 30
+
+
+def prepare_dataset(dataset, window_x=5, window_y=5):
+    Xs = []
+    ys = []
+    for datum in dataset:
+        img = datum['img']
+        mask = datum['mask']
+        for selection, mask_selection in prepare_image(img, mask):
+            Xs.append(selection)
+            ys.append(mask_selection)
+    return np.array(Xs), np.array(ys)
```

Comment:
@mrshu Shouldn't be training data divided further to training and validation data. And then evaluate final score once on test data?